core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -143,8 +173,20 @@ public boolean init(Endpoint endpoint) {
                 runThreadLocalContextCustomizer();
             }
 
+            if (eventLoop == null) {
+                assert factory != null;
+                final ReleasableHolder<EventLoop> releasableEventLoop =
+                        factory.acquireEventLoop(this.endpoint, sessionProtocol());
+                eventLoop = releasableEventLoop.get();
+                log.addListener(log -> releasableEventLoop.release(), RequestLogAvailability.COMPLETE);
```

Comment:
nit: `log.addListener(log -> ...)` -> `log.addListener(ignored -> ...)` to avoid duplication?

Reply:
Fixed. 😄